src/components/MultiMatrix/MultiMatrix.vue

```diff
@@ -726,12 +784,24 @@ export default Vue.extend({
               this.$emit('updateMatrixLegends', true, true);
             }
           } else {
-            this.selectElement(d);
-            this.selectNeighborNodes(d.id, d.neighbors);
+            rowEnter
+              .on('click', (d: Node) => {
+                this.selectElement(d);
+                this.selectNeighborNodes(d.id, d.neighbors);
+              })
+              .attr('cursor', 'pointer');
           }
         });
 
       rowEnter.append('g').attr('class', 'cellsGroup');
+      if (this.enableGraffinity === false) {
+        rowEnter
+          .on('click', (d: Node) => {
+            this.selectElement(d);
+            this.selectNeighborNodes(d.id, d.neighbors);
+          })
```

Comment:
This is now captured on 787
```suggestion
        rowEnter
```